packages/trip-details/src/TripDetails.story.tsx

```diff
@@ -236,6 +237,10 @@ export const EScooterRentalTransitItinerary = makeStory({
   itinerary: eScooterRentalTransiteScooterRentalItinerary
 });
 
+export const OTP2EScooterRentalTransitItinerary = makeStory({
+  itinerary: otp2ScooterItinerary
+});
+
```

Comment:
What is this story supposed to show?

Reply:
This is here for completeness, I guess the main thing I was checking for is that it didn't crash while parsing the slightly different OTP2 itinerary format. If you don't think that's enough of a reason to have a story, we can remove it!

Reply:
Then add a comment saying what you stated.

Reply:
Comment added in 2f4508cf91057c01c2bab6f0ec75666b49d60ecd